extra/mcs_backup_manager.sh

```diff
@@ -2844,62 +2930,248 @@ process_dbrm_backup() {
     if ! $quiet; then printf "[+] Complete\n\n"; fi;
 }
 
-# for next release
+confirm_cmapi_online_and_configured() {
+    cmapi_current_status=$(mcs cmapi is-ready 2> /dev/null);
+    if [ $? -ne 0 ]; then
+
+        # if cmapi is not online - check systemd is running and start cmapi
+            if [ "$(ps -p 1 -o comm=)" = "systemd" ]; then
+            eval $cmapi_start_command
+            sleep 2;
```

Comment:
Probably could be not enough for cmapi becomes ready.
Take a look at https://github.com/mariadb-corporation/mariadb-columnstore-engine/blob/develop/cmapi/check_ready.sh

Reply:
for now im writing a function cmapi_check_ready thats a small augmentation of that script, let me know if you think theres a more maintainable/better way to import it because i could technically run it as it with the function piping to dev/null